addons/custom-block-shape/addon.json

```diff
@@ -0,0 +1,108 @@
+{
+  "name": "Custom block shape",
+  "description": "Adjust the padding, corner radius, and notch height of Scratch blocks.",
+  "tags": ["editor", "theme", "beta"],
+  "credits": [
+    {
+      "name": "SheepTester",
+      "link": "https://sheeptester.github.io/hello-world/userscripts/"
```

Comment:
might be more meaningful to point to https://scratch.mit.edu/users/Sheep_maker/, at least I'll probably change it to that for TW because I don't want links directly to userscripts because of Scratch policy

Reply:
I'll change that -- other addons are linking directly to the userscript, so that's why I chose to include that.